packages/identity-claims/src/index.js

```diff
@@ -56,6 +56,31 @@ export const createIdentityClaimObject = (
   )
 }
 
+/**
+ * Create a valid META Identity Claim object for a profile claim
+ * This is a self-issued claim, usually referencing a Swarm hash of profile data
+ *
+ * @param  {Object} account            Ethereum account object
+ * @param  {String} account.address    Account Ethereum address
+ * @param  {String} account.privateKey Account private key
+ * @param  {String} claimMessage       Raw identity claim message
+ * @param  {String} subProperty        ...
```

Comment:
The above two fields should be clarified as being related to each other, i came to this conclusion having looked at the test spec; it should be clear here.

Reply:
`claimMessage` and `subProperty`?

Looks like I missed out the `subProperty` description anyway! So will add and clarify relationship.

Reply:
Yup, those two.